src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/RequestLogs.java

```diff
@@ -11,11 +11,11 @@ public class RequestLogs {
     @Field("clientRequestId")
     public String clientRequestId;
 
-    @Field("requestId")
-    public String requestId;
+    @Field("gatewayRequestId1")
```

Comment:
Why do we need two `gatewayRequest` Ids?

Reply:
In HIP linking, the facade gets a request to /link-records, that will be the client requestId, now for every request to the gateway there are two requestId which are interacting with ABDM gateway.
![image](https://github.com/NHA-ABDM/ABDM-wrapper/assets/85893494/2c2ca1bd-8450-4802-b900-a0f9e7b2160a)
as described in the documentation